recipes/civetweb/all/conanfile.py

```diff
@@ -82,7 +82,10 @@ def layout(self):
 
     def requirements(self):
         if self.options.with_ssl:
-            self.requires("openssl/1.1.1t")
+            if Version(self.version) < "1.16":
```

Comment:
```
            if Version(self.version) < "1.16":
                self.requires("openssl/1.1.1t")
            else:
                self.requires("openssl/[>=1 <4]")
```

I would suggest something like this - indeed Conan 1.x will not be able to resolve a version range that does not include OpenSSL 3.x - I wouldn't complicate things, if the older versions of civetweb are _not_ compatible with OpenSSL 3.x, we can keep it to the most recent OpenSSL 1.1.1.

OpenSSL 1.1.1 will be on its end of life later this year - https://www.openssl.org/blog/blog/2023/03/28/1.1.1-EOL/ - I think there won't be may more version bumps anyway in this period of a few months until then.

Reply:
done